features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/presequite/SessionReplayRequirement.kt

```diff
@@ -0,0 +1,74 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.internal.presequite
+
+import com.datadog.android.api.InternalLogger
+import okhttp3.internal.toImmutableMap
```

Comment:
we shouldn't use `okhttp3.internal` package, it doesn't exist anymore in OkHttp 5.x, see https://github.com/DataDog/dd-sdk-android/issues/1282

Reply:
It's imported automatically, I will change it to `toMap`